code/__HELPERS/math.dm

```diff
@@ -0,0 +1,107 @@
+/**
+ * Get a list of turfs in a line from `starting_atom` to `ending_atom`.
+ *
+ * Uses the ultra-fast [Bresenham Line-Drawing Algorithm](https://en.wikipedia.org/wiki/Bresenham%27s_line_algorithm).
+ */
+/proc/get_line(atom/starting_atom, atom/ending_atom)
```

Comment:
proc i got from TG because its usefull

Reply:
Mmmm this is some good alphabet soup.

~im not asking anything just chuckling~